adam_preprocessing/shape_stroke_graph_learner.py

```diff
@@ -161,16 +163,54 @@ def main():
                 topk=(1,),
             )[0]
         )
+        situation_test_acc = get_situation_accuracy(
+            nn.LogSoftmax(dim=1)(
```

Comment:
I don't think there's any reason to take the softmax before inputting to `get_situation_accuracy()`. It should work correctly directly on the logits

Reply:
Fair enough.

Reply:
Removed the softmax.